employee.rb

```diff
@@ -0,0 +1,30 @@
+class Employee
+  attr_accessor :name, :salary, :reviews, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+    @satisfactory = []
```

Comment:
You set satisfactory to an array here, but you use it as a true/false elsewhere.  It would probably be best to leave off this line to not confuse future developers (including yourself).